tests/invariances.py

```diff
@@ -13,6 +16,97 @@ def assert_potential_invariance(energy_fn, x0, params, box=None):
         check_grads(energy_fn, (x0, params, box), order=1, eps=1e-5)
         check_grads(energy_fn, (x0, params, box), order=2, eps=1e-7)
 
-    # (ytz: TODO) test translational and rotational invariance of
+    # test translational and rotational invariance of
     # energy and its derivatives, with and without box
-    
\ No newline at end of file
+    energy = energy_fn(x0,params,None)
+    force_fn = jax.grad(energy_fn,argnums=(0,))
+    forces = force_fn(x0,params,None)[0]
+    dEdp = jax.jacfwd(energy_fn,argnums=(1,))
+    d2Edxdp = jax.jacfwd(dEdp,argnums=(0,))
+    dparam = dEdp(x0,params,None)[0]
+    mixed = d2Edxdp(x0,params,None)[0][0]
+
+    for _ in range(3):
+        trans_vector = np.random.rand(3)
+        trans_x = x0 + trans_vector
+        trans_energy = energy_fn(trans_x,params,None)
+        trans_forces = force_fn(trans_x,params,None)[0]
+        trans_dEdp = dEdp(trans_x,params,None)[0]
+        trans_mixed = d2Edxdp(trans_x,params,None)[0][0]
+        np.testing.assert_almost_equal(trans_energy,energy,decimal=5)
```

Comment:
How many decimal places do they agree to? I know you set 5 here but what's the actual agreement?

Reply:
Bonded terms looked like they were generally accurate to unit roundoff. Nonbonded terms failed when testing to the default 6 decimal places but passed when changing to 5 decimal places

Reply:
1e-5 seems a little too high, numerical roundoff should be around 1e-13. Are the rotation matrices themselves in 64-bit?  You may need to cast them via .astype(dtype=np.float64) before using them.

Reply:
I'm not sure it's due to the data type of the rotation matrix since all of the bonded terms seem to be accurate to around 12 digits even for rotations

Reply:
Looks like the first major difference (~1e-5) is in the Lennard-Jones energy calculations - maybe the number of fp operations in the LJ calculations are scaling up the error?